CodeStarterCamp_Week1/main.swift

```diff
@@ -2,27 +2,55 @@
 //  main.swift
 //  CodeStarterCamp_Week1
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by Hoon on 2023/02/23.
 //
 
 import Foundation
 
-func makeIceCreamBody() {
-    let iceCreamParts:String = "***********" //아이스크림 몸통 만드는 함수
-    for _ in 1...8 {
-        print(iceCreamParts)
+
+//길이에 따른 막대를 그리는 함수
+func makeBarLength(lengthBar:Int) {
```

Comment:
```suggestion
func makeBarLength(lengthBar: Int) {
```

보통 매개변수 명과 타입을 표기하는 컨벤션은 이렇습니다! 
이 부분은 [Functions](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions#Defining-and-Calling-Functions)의 공식문서의 예제를 쭉 보시면 도움이 되실 겁니다! 
아래 코드 또한 공식문서의 컨벤션으로 맞춰주세요!